resource_migrations/migration_1.py

```diff
@@ -0,0 +1,67 @@
+from time import sleep
+
+from kubernetes import client
+from kubernetes import config as k8s_config
+
+from run_all import parse_args
+
+
+def adjust_annotations(args):
+    k8s_config.load_config()
+    k8s_api = client.CustomObjectsApi(client.ApiClient())
+
+    jss = k8s_api.list_namespaced_custom_object(
```

Comment:
The API supports pagination, so I think it would be good to use it (probably with a rather large page size).

Reply:
I am using pagination now.